src/Components/Facility/FacilityCreate.tsx

```diff
@@ -39,81 +34,80 @@ import {
   getWardByLocalBody,
 } from "../../Redux/actions";
 import * as Notification from "../../Utils/Notifications.js";
-import {
-  ErrorHelperText,
-  MultilineInputField,
-  PhoneNumberField,
-  SelectField,
-  TextInputField,
-} from "../Common/HelperInputFields";
+import { ErrorHelperText, PhoneNumberField } from "../Common/HelperInputFields";
 import GLocationPicker from "../Common/GLocationPicker";
 import { goBack } from "../../Utils/utils";
 import useWindowDimensions from "../../Common/hooks/useWindowDimensions";
 import MultiSelectMenuV2 from "../Form/MultiSelectMenuV2";
+import TextInputFieldV2 from "../Common/components/TextInputFieldV2";
```

Comment:
@cp-Coder use `TextFormField` instead and mark `TextInputFieldV2` as deprecated. From here on `TextFormField` will be maintained.

Reply:
Ref: https://github.com/coronasafe/care_fe/pull/4309